### DISCUSSION TITLE
PNP Modern Search Web Parts - Query Text

### DISCUSSION THREAD

**Discussion by jenniferpigeon** [microsoft-search/pnp-modern-search#2878] [Q&A] [answered]
Hello - I am on version 4.8.0.0 and here is what I am trying to accomplish:

1. Limit my search to one subsite 
2. Return all results from that subsite based on the search term entered.

In the Query template, if I enter "{searchTerms}" with nothing else, I am able to return all results from my entire HUB based on the search term entered.
If I enter {searchTerms} Path:https://xxxx.sharepoint.com/sites/sitename/subsitename/ and nothing else, I am able to limit my results to only my subsite.

Question: How can I combine the queries to perform both actions?: limit my search results to one subsite and return all results from that subsite based on the search term entered.

Note: I want the results to also show the search term in bold when delivering the results.  For some reason, when I limit my search to my subsite only, I get results with no bolded search terms.

Thank you.

**Comment by kasperbolarsen** [comment]
{SearchTerms} path:https://xxxx.sharepoint.com/sites/sitename/subsitename/*


Should be the right query. Your Results web part is connected to the search box,  right?

**Comment by jenniferpigeon** [comment, marked as answer]
Yes, my results web part is connected to the search box.  The query you sent returns results from my subsite, but it does not bold my search term in the results.

**Reply by kasperbolarsen** [reply]
Does Path:https://xxxx.sharepoint.com/sites/sitename/subsitename/* give a result in either the classic searchcenter or in ms search?

**Comment by jenniferpigeon** [comment]
Hmm, my subsites do not seem to appear in search results.  I checked several and none of them return any results.

**Reply by jenniferpigeon** [reply]
This is now working.  Thanks for the help!

**Reply by kasperbolarsen** [reply]
Can you share the reason for the issue for future reference?

**Reply by jenniferpigeon** [reply]
I am not sure why, but inputting Path:https://xxxx.sharepoint.com/sites/sitename/subsitename/* started working.
